onnxruntime/core/providers/nnapi/nnapi_builtin/builders/model_builder.cc

```diff
@@ -605,20 +623,27 @@ Status ModelBuilder::Compile(std::unique_ptr<Model>& model) {
   return Status::OK();
 }
 
-int32_t ModelBuilder::FindActivation(const Node& node, const NodeArg& output) {
+int32_t ModelBuilder::FindActivation(const NodeUnit& node_unit, const NodeArg& output) {
+  (void)node_unit;
```

Comment:
nit: ORT_UNUSED_PARAMETER is the more standard way

Reply:
Forgot to remove